cmd/storagenode/Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG GOARCH
 ENV GOARCH ${GOARCH}
 EXPOSE 28967
 WORKDIR /app
-COPY resources/certs.pem /etc/ssl/certs/ca-certificates.crt
```

Comment:
I'm wondering what this certificate in our repository was about. It was a copy of the `ca-certificates.crt`?

Reply:
As answered in standup, this was a single root CA of lets encrypt that was needed in the past. We entirely overwrote the default CA list.